src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import './Calculator.css';
+
+function Calculator() {
```

Comment:
- [ ] Your _Calculator_ component _(Calculator.js)_ is not classed based. Kindly change this component to a class-based component.

Reply:
Done